core/system/src/mpu/vmpu_freescale_k64_mem.c

```diff
@@ -301,11 +303,11 @@ void vmpu_mem_init(void)
     rgd = g_mem_box[0].acl;
     /* Copy ACLs [0, 1] into MPU regions [_1_, _2_]. Region 0 is the background region! */
     for(t = 1; t < K64F_MPU_REGIONS_STATIC; t++, rgd++) {
-        word = MPU->WORD[t];
-        word[0] = rgd->word[0];
-        word[1] = rgd->word[1];
-        word[2] = rgd->word[2];
-        word[3] = 1;
+        region = (MPU_Region *) MPU->WORD[t];
+        region->STARTADDR   = rgd->start_addr;
```

Comment:
Do not align the `=` signs as it becomes a pain to keep them aligned later on when we change this code :)